docs/paper/admin/misc/paper-bug-fixes.md

```diff
@@ -0,0 +1,64 @@
+---
+slug: /misc/paper-bug-fixes
+---
+
+# Paper Bug Fixes
+
+Paper fixes gameplay and mechanics inconsistencies. The most prevalent fixes being TNT Duplication and Bedrock Breaking.
+
+## Vanilla Bug Fixes
+
+Paper fixes many vanilla bugs that were not intended by Mojang. These bugs are patched to fix behavior or prevent abuse and
+instability on the server. Many of our fixes are configurable as we understand that some servers may want to keep the
```

Comment:
```suggestion
instability on the server. Many of our fixes are configurable, as we understand that some servers may want to keep the
```

Reply:
I'm not sure we want to use the word `Many` here to describe the number of configurable fixes. I would say most aren't configurable. I think `Some` is better.